ape_vyper/compiler.py

```diff
@@ -95,6 +100,22 @@ def get_pragma_spec(source: Union[str, Path]) -> Optional[SpecifierSet]:
         return None
 
 
+def get_optimization_pragma(source: Union[str, Path]) -> Union[str, bool]:
```

Comment:
just return `None` if not set
```suggestion
def get_optimization_pragma(source: Union[str, Path]) -> Optional[str]:
```